brc20/verifier.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/ordinox/btc-service/config"
 )
 
+// Return the sender and reciever of a BRC20 transfer
```

Comment:
nit: would be nice to type cast these strings and have the ret val as sender/recvr appropriately - instead of this comment